charts/neuvector/latest/questions.yml

```diff
@@ -0,0 +1,11 @@
+questions:
+- variable: registry
+  default: "docker.io"
+  description: image registry
+  type: string
+  label: Image Registry
+- variable: imagePullSecrets
```

Comment:
`required: true` ?

Reply:
Like I explained above, pull secrete is not a required parameter.